### DISCUSSION TITLE
Add Quasar to Nuxt.

### DISCUSSION THREAD

**Discussion by thebrownbox** [quasarframework/quasar#10900] [General - Components / Directives / etc]
Hi guys,
I have used Quasar for my VueJS app.
Now I move to Nuxt project, I want to add Quasar to project too.
**Is that possible?**
(I have try some example but it didn't work)

**Comment by metalsadman** [comment]
yes, use quasar as a [vue cli plugin ](https://quasar.dev/start/vue-cli-plugin#introduction).

**Reply by thebrownbox** [reply]
Hi @metalsadman ,
Thank you for your response.
I should create Nuxt project using 'npx create-nuxt-app' or with vue-cli (vue create-app):
I have tried both ways, but both ways are not working...

**Option1:  With create-nuxt-app,:**
I did as below steps: [link](https://github.com/thebrownbox/nuxt-with-quasar/tree/main/create-nuxt-app-quasar-1.16.0)

**Option 2: With vue-cli**
I did as below steps: [link](https://github.com/thebrownbox/nuxt-with-quasar/tree/main/create-nuxt-from-vue-cli)

Would you mind to take a look to see which step that I made wrong?

Thank you very much....

**Reply by ontwikkelfabriek** [reply]
@thebrownbox 

A quick look at your second github link. You forgot to install the node modules with the  `yarn` command.

Also see this thread:
https://stackoverflow.com/questions/55997850/error-in-running-nuxt-project-nuxt-is-not-recognized-as-an-internal-or-exter

**Reply by thebrownbox** [reply]
Hi @ontwikkelfabriek ,
Thank you for your response.
But I'm not sure I understand you correctly or not!
I have installed every packages step by step as in the README file....

**Reply by ontwikkelfabriek** [reply]
@thebrownbox 

No you did **not**, at least not if you followed your own steps that you wrote down. The `yarn` or `yarn install` command is for installing deps and devdeps stated in the package.json ( that's where the `nuxt` dep is defined).  That's probably why your get :
`yarn dev => Not recognize **nuxt** command`

Did you see this guide btw?:
https://github.com/quasarframework/vue-cli-plugin-quasar

**Comment by DrScorpion86** [comment]
For those who were also looking for an answer... for Nuxt 3.
[nuxt-quasar-ui](https://www.npmjs.com/package/nuxt-quasar-ui)

**Reply by UkonnRa** [reply]
What this plugin contains? I am trying to reduce the third-party dependencies as much as possible.

If I consider Quasar only as a component library, like Vuetify or MUI, and do not use the Quasar SSR part, only use Nuxt to handle all the SSR related issue.

Can I have a simpler integration solution?
